### PR TITLE
DOCSP-40542 Compass Quit Functionality Update

### PR DESCRIPTION
**Pull request by ianf-mongodb**
## DESCRIPTION

- Add quit setting to the settings page.
- Updated all keyboard short cuts to use `:kbd:` role.
- Updated the close window hotkey to be cmd + shift + w to reflect recent app change.
- Add ref to the keyboard shortcuts page.

## STAGING

https://preview-mongodbianfmongodb.gatsbyjs.io/compass/DOCSP-40542/settings/settings-reference/#settings

https://preview-mongodbianfmongodb.gatsbyjs.io/compass/DOCSP-40542/keyboard-shortcuts/

## JIRA

https://jira.mongodb.org/browse/DOCSP-40542

## BUILD LOG

https://workerpool-boxgs.mongodbstitch.com/pages/job.html?collName=queue&jobId=6684128efe162e50061300a9

## Self-Review Checklist

- [ ] Is this free of any warnings or errors in the RST?
- [ ] Is this free of spelling errors?
- [ ] Is this free of grammatical errors?
- [ ] Is this free of staging / rendering issues?
- [ ] Are all the links working?

## External Review Requirements

[What's expected of an external reviewer?](https://wiki.corp.mongodb.com/display/DE/Reviewing+Guidelines+for+the+MongoDB+Server+Documentation)

### CODE REVIEW

File: source/settings/settings-reference.txt [hunk 1]
```diff
@@ -107,6 +107,15 @@ You can configure the following settings on the |compass| interface:
        enabled. You cannot toggle this option on Linux. You can toggle this 
        setting only on Windows or macOS.
 
+   * - Show Quit Confirmation Dialog
+     - General
+     - Toggle whether a confirmation dialog is shown when quitting Compass.
+       Checking the ``Do not ask again`` checkbox on the quit dialog 
+       disables this setting.
```
Comment (sarah-olson-mongodb) [inline source/settings/settings-reference.txt:114]: (structure)
To disable this setting, check "Do not ask again".

File: source/settings/settings-reference.txt [hunk 2]
```diff
@@ -107,6 +107,15 @@ You can configure the following settings on the |compass| interface:
        enabled. You cannot toggle this option on Linux. You can toggle this 
        setting only on Windows or macOS.
 
+   * - Show Quit Confirmation Dialog
+     - General
+     - Toggle whether a confirmation dialog is shown when quitting Compass.
```
Comment (sarah-olson-mongodb) [inline source/settings/settings-reference.txt:112]: (term and active voice)
I'm a little concerned about toggle. I think we could get more specific. Also we want to stay away from "is shown". So maybe something like:
Specifies whether to display the confirmation dialog when you exit Compass.

File: source/keyboard-shortcuts.txt [hunk 3]
```diff
@@ -1,9 +1,9 @@
+.. _compass-keyboard-shortcuts:
+
 ==================
 Keyboard Shortcuts
 ==================
```
Comment (sarah-olson-mongodb) [inline source/keyboard-shortcuts.txt:5]: (praise)
Nice improvements. Thanks for doing that!

**Review by sarah-olson-mongodb — APPROVED** [review]
Thanks @ianf-mongodb! LGTM with a few items for consideration.

File: source/settings/settings-reference.txt (hunk 1 above)
Comment (ianf-mongodb) [reply source/settings/settings-reference.txt:114]: This removes the reference to the quit dialog box which is a separate element than where the setting is stored. This is a supplemental way to turn off the setting option. 

So it needs to be:

"You can also check the ``Do not ask again`` checkbox on the quit dialog to disable the setting."

**Comment by ianf-mongodb** [comment]
@Anemy this is ready for review.

File: source/settings/settings-reference.txt [hunk 4]
```diff
@@ -107,6 +107,15 @@ You can configure the following settings on the |compass| interface:
...
        setting only on Windows or macOS.
 
+   * - Show Quit Confirmation Dialog
+     - General
+     - Toggle whether a confirmation dialog shows when quitting Compass.
+       You can also check the ``Do not ask again`` checkbox on the quit 
+       dialog to disable this setting.
+
+       You can also toggle the quit dialog with the keyboard shortcut:
+       :kbd:`ctrl` + :kbd:`Q` on Windows or :kbd:`cmd` + :kbd:`Q` on Mac.
```
Comment (Anemy) [inline source/settings/settings-reference.txt:117]: Maybe:
```suggestion
       The quit dialog is also shown when quitting with the keyboard shortcut:
       :kbd:`ctrl` + :kbd:`Q` on Windows or :kbd:`cmd` + :kbd:`Q` on Mac.
```

**Review by Anemy — APPROVED** [review]

**Review by Anemy — APPROVED** [review]

**Pull request merged**